src/webloader.js

```diff
@@ -6,9 +6,13 @@ import { modes } from './modes/index.js';
 import { setCookie, getCookie } from "./cookies.js";
 
 
+function getId(evolv, key) {
```

Comment:
This should return the id -- right now it returns the context

Reply:
Fixed - also found a bug where we are checking if the `key` exists in `evolv.context` and then using `evolv[key]` to get the `id` - should be `evolv.context[key]`